server/sonar-server/src/main/resources/org/sonar/server/project/ws/search-example.json

```diff
@@ -10,14 +10,16 @@
       "id": "project-uuid-1",
       "key": "project-key-1",
       "name": "Project Name 1",
-      "qualifier": "TRK"
+      "qualifier": "TRK",
+      "visibility": "public"
     },
     {
       "organization": "my-org-1",
       "id": "project-uuid-2",
       "key": "project-key-2",
       "name": "Project Name 1",
-      "qualifier": "TRK"
+      "qualifier": "TRK",
+      "visibility": "public"
```

Comment:
Examples do not have to be _complete_ in my opinion. (But of course variety is a bonus)